README.md

```diff
@@ -189,3 +189,92 @@ GET /v1/properties/:propertyReference/
   ]
 }
 ```
+
+## Create a new repair
+
+Create a repair request, with or without a list of work orders
+
+```
+POST /v1/properties/:id/repairs
```

Comment:
Note: This URL is an idea @leeky and I have been discussing. 

It makes sense on the basis that internal repairs are always for a property, and one can imagine `GET /v1/properties/:property_reference/repairs` returning all the repairs for a particular property at some point in the future. 

However, individual repairs don't need to be fetched in this scoped way: we only need to pass a repair reference:

```
GET /v1/repairs/:repair_request_reference
```
 and not the property reference as well - e.g.
```
GET /v1/properties/:property_reference/repairs/:repair_request_reference
```

I don't think we need/want to add that restriction.

Reply:
This makes sense. If you already have a reference of a specific object you should not need to specify another.